AllReadyApp/Web-App/AllReady/Views/Activity/Activity.cshtml

```diff
@@ -292,13 +301,14 @@
                         tasks = Model.Tasks,
                         skills = Model.RequiredSkills,
                         userSkills = Model.UserSkills,
-                        isVolunteeredForActivity = Model.IsUserVolunteeredForActivity
+                        isVolunteeredForActivity = Model.IsUserVolunteeredForActivity,
+                        signupModelSeed = Model.SignupModel
                     });
 
-
-        $('#phone-input').mask("(999) 999-9999");
-            
-    </script>
+        </script>
+    <script type="text/javascript" src="~/js/ko.bindings.js"></script>
+    <script type="text/javascript" src="~/lib/knockout.mapping/knockout.mapping.js"></script>

```

Comment:
@mheggeseth the reason for this is that when I went to bower to try to find the  knockout-mapping plugin I found three entries, all of which were forks of Steve Sanderson's "official" plugin.  The three that I found are [here](https://github.com/crissdev/knockout.mapping), [here](https://github.com/LRPalacios/knockout.mapping) and [here](https://github.com/crissdev/knockout.mapping).  Steve Sanderson's version is [here](https://github.com/SteveSanderson/knockout.mapping).  So, instead of using one of the forks i just downloaded Sanderson's.

Do you want me to swtch to one of the forks or just continue with the downloaded version?